package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "set-protocol-contracts",
-  "version": "1.2.1-beta.13",
+  "version": "1.2.1-beta.14",
   "description": "Smart contracts for {Set} Protocol",
   "main": "dist/artifacts/index.js",
```

Comment:
Make sure to update `delete-dev-addresses` and also the gitignore might need to be updated?